backend/test/richard_burton_web/controllers/publication_controller_test.exs

```diff
@@ -666,11 +666,39 @@ defmodule RichardBurtonWeb.PublicationControllerTest do
       assert publication |> Map.take(Map.keys(update_attrs)) == update_attrs
     end
 
+    test "handles nested attributes updates", %{conn: conn, path: path, publication: publication} do
+      data = %{countries: "GB, UY"}
+      resp = conn |> put(path, data)
+
+      assert resp.status == 200
+
+      assert publication
+             |> Repo.reload()
+             |> Publication.preload()
+             |> Map.get(:countries)
+             |> Enum.map(&Map.get(&1, :code))
+             |> Enum.sort()
+             |> Enum.join(", ") == data.countries
```

Comment:
NIT: we could move the pipeline result to a variable to enhance readability, the `== data.countries` is kind of lost at first sight

Reply:
Makes sense